src/stores/network.ts

```diff
@@ -1,9 +1,9 @@
 import { reactive } from 'vue'
-import * as Nimiq from '@nimiq/core'
+import initClient, { Client, ClientConfiguration, KeyPair } from '@nimiq/core/web'
```

Comment:
You should be able to still do `* as Nimiq`, and then `initClient()` becomes `Nimiq.default()`.

Reply:
Right. 
What do you prefer Janine?

I like the way in PR more, as it explicitly imports only what is needed, but maybe you have a different opinion